src/lib/model/account/utils.ts

```
@@ -7,14 +7,19 @@ export const calBonded = (
 ) => {
   if (!totalDelegations || !totalUnbondings) return undefined;
 
-  return Object.keys(totalDelegations).reduce<Record<string, TokenWithValue>>(
-    (total, denom) => ({
+  const totalBonded = Object.entries(totalDelegations).reduce<
+    Record<string, TokenWithValue>
+  >(
+    (total, [denom, token]) => ({
       ...total,
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)